recipes/CommonVoice/ASR/CTC/hparams/train_de_with_wav2vec.yaml

```diff
@@ -0,0 +1,177 @@
+# Model: wav2vec2 + DNN + CTC
+# Augmentation: SpecAugment
+# Authors: Titouan Parcollet 2021
+#          Mirco Ravanelli 2021
+#          Sangeet Sagar 2022 (minor edits)
+# Language: German (DE)
+# ################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 8200
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !ref results/train_wav2vec2_char/<seed>
+wer_file: !ref <output_folder>/wer.txt
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# Dataset prep parameters
+data_folder: !ref /ds/audio/CommonVoiceCorpus10.0/cv-corpus-10.0-2022-07-04/de
+train_tsv_file: !ref <data_folder>/train.tsv
+dev_tsv_file: !ref <data_folder>/dev.tsv
+test_tsv_file: !ref <data_folder>/test.tsv
+accented_letters: True
+language: de
+skip_prep: False
+
+# Data files
+train_csv: !ref <save_folder>/train.csv
+valid_csv: !ref <save_folder>/dev.csv
+test_csv: !ref <save_folder>/test.csv
+
+# We remove utterance slonger than 10s in the train/dev/test sets as
+# longer sentences certainly correspond to "open microphones".
+avoid_if_longer_than: 10.0
+
+# Training parameters
+number_of_epochs: 45
+lr: 1.0
+lr_wav2vec: 0.0001
+sorting: ascending
+auto_mix_prec: False
+sample_rate: 16000
+ckpt_interval_minutes: 30 # save checkpoint every N min
+
+# With data_parallel batch_size is split into N jobs
+# With DDP batch_size is multiplied by N jobs
+# Must be 6 per GPU to fit 16GB of VRAM
+batch_size: 8
+test_batch_size: 8
+num_workers:  8
```

Comment:
Maybe we should have two different num_workers: one for the dataloader_options and another for test_dataloader_option

Reply:
Fixed
```yaml
dataloader_num_workers:  8
test_num_workers:  8

dataloader_options:
   batch_size: !ref <batch_size>
   num_workers: !ref <dataloader_num_workers>
test_dataloader_options:
   batch_size: !ref <test_batch_size>
   num_workers: !ref <test_num_workers>
```